File: Programmers/Level1/음양더하기/음양더하기.py
def solution(absolutes, signs):
    return sum([absolutes[i] if signs[i] == True else absolutes[i] * (-1) for i in range(len(absolutes))])

absolutes = [4, 7, 12]
signs = [True, False, True]

print(solution(absolutes, signs))

File: Programmers/Level1/음양더하기/음양더하기.js
function solution(absolutes, signs) {
  return absolutes.reduce(
    (acc, cur, idx) => (signs[idx] ? acc + cur : acc - cur),
    0
  );
}

const absolutes = [4, 7, 12];
const signs = [true, false, true];

console.log(solution(absolutes, signs));
